Samples/word-citation-management/README.md

```diff
@@ -0,0 +1,124 @@
+---
+page_type: sample
+urlFragment: word-citation-management
+products:
+- office-word
+- m365
+languages:
+- javascript
+extensions:
+  contentType: samples
+  technologies:
+  - Add-ins
+  createdDate: 11/15/2023 4:00:00 PM
+description: "Shows how to manage citations in a Word document."
+---
+
+# Manage citations in a Word document
+
+## Summary
+
+This sample shows how to manage citations in Word with an add-in. The add-in uses the [insertEndnote API](https://learn.microsoft.com/javascript/api/word/word.range#word-word-range-insertendnote-member(1)) to list inserted citations at the end of the document.
+
+For documentation related to this sample, see [Citation management](https://learn.microsoft.com/office/dev/add-ins/word/citation-management).
```

Comment:
@AlexJerabek I was thinking this would be for folks who came directly here from search. I'll move it to the end of the sample so it's still available for completeness